conf/irida_next.config

```diff
@@ -18,83 +18,191 @@ iridanext {
+                    "meta.downsampled"                               : "Downsampled",
+                    "SpeciesTopHit"                                  : "Species",
+                    "ECTyperSubtyping.0.Database"                    : "ECTyper Database",
+                    "ECTyperSubtyping.0.Evidence"                    : "ECTyper Evidence",
+                    "ECTyperSubtyping.0.GeneCoverages(%)"            : "ECTyper GeneCoverages (%)",
+                    "ECTyperSubtyping.0.GeneIdentities(%)"           : "ECTyper GeneIdentities (%)",
+                    "ECTyperSubtyping.0.GeneScores"                  : "ECTyper GeneScores",
+                    "ECTyperSubtyping.0.H-type"                      : "ECTyper H-Antigen",
+                    "ECTyperSubtyping.0.O-type"                      : "ECTyper O-Antigen",
+                    "ECTyperSubtyping.0.QC"                          : "ECTyper QCFlag",
+                    "ECTyperSubtyping.0.Serotype"                    : "ECTyper Serotype",
+                    "ECTyperSubtyping.0.Species"                     : "ECTyper Subtyping",
+                    "ECTyperSubtyping.0.Warnings"                    : "ECTyper Warnings",
+                    "LISSEROSubtyping.0.SEROTYPE"                    : "LISSERO Serotype",
+                    "QUAST.0.GC (%)"                                 : "GC (%)",
+                    "RawReadSummary.R1.mean_sequence_length"         : "Mean Sequence Length Forward",
+                    "SISTRSubtyping.0.cgmlst_ST"                     : "SISTR cgMLST ST",
+                    "SISTRSubtyping.0.cgmlst_found_loci"             : "SISTR cgMLST Found Loci",
+                    "SISTRSubtyping.0.cgmlst_genome_match"           : "SISTR cgMLST Genome Match",
+                    "SISTRSubtyping.0.cgmlst_matching_alleles"       : "SISTR cgMLST Matching Alleles",
+                    "SISTRSubtyping.0.cgmlst_subspecies"             : "SISTR cgMLST Subspecies",
+                    "SISTRSubtyping.0.h1"                            : "SISTR H1",
+                    "SISTRSubtyping.0.h2"                            : "SISTR H2",
+                    "SISTRSubtyping.0.o_antigen"                     : "SISTR Antigen",
+                    "SISTRSubtyping.0.qc_messages"                   : "SISTR QC Message",
+                    "SISTRSubtyping.0.qc_status"                     : "SISTR QC Status",
+                    "SISTRSubtyping.0.serogroup"                     : "SISTR Serogroup",
+                    "SISTRSubtyping.0.serovar"                       : "SISTR Serovar",
+                    "SISTRSubtyping.0.serovar_antigen"               : "SISTR Serovar Antigen",
+                    "SISTRSubtyping.0.serovar_cgmlst"                : "SISTR Serovar cgMLST",
+                    "SeqtkBaseCount"                                 : "BaseCount",
+                    "SevenGeneMLSTReport.0.alleles.abcZ"             : "abcZ",
```

Comment:
Are the alleles listed specific to the PNC organisms?

Reply:
These are based on what was requested to keep for metadata and are specific to PNC organisms.